isis/src/base/objs/PvlObject/PvlObject.h

```diff
@@ -76,6 +79,7 @@ namespace Isis {
       PvlObject();
       PvlObject(const QString &name);
       PvlObject(const PvlObject &other);
+      PvlObject(const QString &name, const nlohmann::json &jsonobj);
```

Comment:
We should not do using statements in the header files because it can pollute downstream. We should either have using json.... in each cpp file or no where at all.